pkg/features/features.go

```diff
@@ -17,5 +18,5 @@ var operatorControllerFeatureGates = map[featuregate.Feature]featuregate.Feature
 var OperatorControllerFeatureGate featuregate.MutableFeatureGate = featuregate.NewFeatureGate()
 
 func init() {
-	OperatorControllerFeatureGate.Add(operatorControllerFeatureGates)
+	utilruntime.Must(OperatorControllerFeatureGate.Add(operatorControllerFeatureGates))
```

Comment:
@everettraven is it fair? Or do we need better error handling here?

Reply:
I think that is fair. Good catch - I totally missed that it returned an error. Would probably be good to fix this in catalogd as well (and add linting there :))

Reply:
@everettraven here is the PR for catalogd - https://github.com/operator-framework/catalogd/pull/98